File: JavaScript/8js 1/correctTail.js
/*
|--------------------------------------------------------------------------
| Is this my tail?
|--------------------------------------------------------------------------
|
*/

function correctTail(body, tail) {
   return body[body.length - 1] == tail
}


function correctTail(body, tail) {
   return body.substr(-1) == tail;
}


function correctTail(body, tail) {
   return body.split("").reverse()[0] === tail;
}


function correctTail(body, tail) {
   return body.slice(-1) == tail
}


console.log(correctTail("Fox", "x"), true);
console.log(correctTail("Rhino", "o"), true);
console.log(correctTail("Meerkat", "t"), true);

File: Python/8py 1/correct_tail.py
"""
|--------------------------------------------------------------------------
| Is this my tail?
|--------------------------------------------------------------------------
|
"""

def correct_tail(body, tail):
   return body[-1] == tail


def correct_tail(body, tail):
   return body.endswith(tail)


def correct_tail(body, tail):    
   return body[len(body)-1:] == tail


print(correct_tail("Fox", "x"), True)
print(correct_tail("Rhino", "o"), True)
print(correct_tail("Meerkat", "t"), True)
print(correct_tail("Emu", "t"), False)
print(correct_tail("Badger", "s"), False)
print(correct_tail("Giraffe", "d"), False)